benchmarks/dynamo/ci_expected_accuracy/update_expected.py

```diff
@@ -0,0 +1,140 @@
+"""
+Update commited CSV files used as reference points by dynamo/inductor CI.
+
+Currently only cares about graph breaks, so only saves those columns.
+
+Hardcodes a list of job names and artifacts per job, but builds the lookup
+by querying github sha and finding associated github actions workflow ID and CI jobs,
+downloading artifact zips, extracting CSVs and filtering them.
+
+Usage:
+
+1) get a rocksdp API key and save it in ~/rocks_api_key
+2) python benchmarks/dynamo/ci_expected_accuracy.py <sha of pytorch commit that has completed inductor benchmark jobs>
+
+Known limitations:
+- doesn't handle 'retry' jobs in CI, if the same hash has more than one set of artifacts, gets the first one
+- needs rocks API key (plan to use a public endpoint and remove this limitation)
+"""
+
+import argparse
+import os
+import urllib
+from io import BytesIO
+from urllib.request import urlopen
+from zipfile import ZipFile
+
+import pandas as pd
+
+from rockset import RocksetClient
+
+
+def query_job_sha(repo, sha, api_key):
+    rs = RocksetClient(api_key=api_key, host="https://api.usw2a1.rockset.com")
+
+    params = list()
+    params.append({"name": "repo", "type": "string", "value": repo})
+    params.append({"name": "sha", "type": "string", "value": sha})
+
+    response = rs.QueryLambdas.execute_query_lambda(
+        query_lambda="commit_jobs_query",
+        version="cc524c5036e78794",
+        workspace="commons",
+        parameters=params,
+    )
+    return response.results
+
+
+def parse_job_name(job_str):
+    return (part.strip() for part in job_str.split("/"))
+
+
+def parse_test_str(test_str):
+    return (part.strip() for part in test_str[6:].strip(")k").split(","))
+
+
+S3_BASE_URL = "https://gha-artifacts.s3.amazonaws.com"
+
+
+def get_artifacts_urls(results, suites):
+    urls = {}
+    for r in results:
+        if "inductor" == r["workflowName"] and f"test" in r["jobName"]:
```

Comment:
nit (linter would complain if run on this)

```suggestion
        if "inductor" == r["workflowName"] and "test" in r["jobName"]:
```